### PR TITLE
[INJIMOB-785]: add location permission rational

### PR DESCRIPTION
**Pull request by srikanth716**

### CODE REVIEW

File: screens/Scan/ScanScreen.tsx [hunk 1]
```diff
@@ -144,6 +144,51 @@ export const ScanScreen: React.FC = () => {
     );
   }
 
+  function LocalPermissionRational() {
+    return (
+      <Overlay
```
Comment (swatigoel) [inline screens/Scan/ScanScreen.tsx:149]: Extract it as separate component

File: machines/bleShare/scan/scanMachine.ts [hunk 2]
```diff
@@ -133,6 +132,8 @@ const model = createModel(
       FACE_VERIFICATION_CONSENT: (isConsentGiven: boolean) => ({
         isConsentGiven,
       }),
+      ALLOW: () => ({}),
```
Comment (swatigoel) [inline machines/bleShare/scan/scanMachine.ts:135]: rename Allow to Allowed

File: screens/Scan/ScanScreenController.ts [hunk 3]
```diff
@@ -73,6 +76,12 @@ export function useScanScreen() {
   type ScanScreenNavigation = NavigationProp<MainBottomTabParamList>;
   const navigation = useNavigation<ScanScreenNavigation>();
   const GOTO_HOME = () => navigation.navigate(BOTTOM_TAB_ROUTES.home);
+  const ALLOW = () => scanService.send(ScanEvents.ALLOW());
```
Comment (swatigoel) [inline screens/Scan/ScanScreenController.ts:79]: here also rename allow to allowed

**Review by swatigoel — CHANGES_REQUESTED** [review]

**Review by swatigoel — APPROVED** [review]

File: screens/Scan/ScanScreen.tsx (hunk 1 above)
Comment (srikanth716) [reply screens/Scan/ScanScreen.tsx]: created screens/Scan/LocationPermissionRational.tsx

File: machines/bleShare/scan/scanMachine.ts (hunk 2 above)
Comment (srikanth716) [reply machines/bleShare/scan/scanMachine.ts]: changed

File: screens/Scan/ScanScreenController.ts (hunk 3 above)
Comment (srikanth716) [reply screens/Scan/ScanScreenController.ts]: changes

**Review by swatigoel — APPROVED** [review]

**Pull request merged**